src/test/java/com/google/cloud/spanner/jdbc/it/ITJdbcSqlScriptTest.java

```diff
@@ -1,282 +0,0 @@
-/*
```

Comment:
Note to other reviwers: this file `ITJdbcSqlScriptTest.java` seems to have been generalized as `ITJdbcScriptTest.java` and github is not able to detect the diff.